<ui version="4.0" >
 <class>SmoYuvPlayer</class>
 <widget class="QWidget" name="SmoYuvPlayer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>370</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <widget class="QGroupBox" name="ControlGroupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>160</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QWidget" name="" >
    <property name="geometry" >
     <rect>
      <x>21</x>
      <y>11</y>
      <width>264</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="PlayButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="dvrcp.qrc" >:/images/play.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PauseButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="dvrcp.qrc" >:/images/pause.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="dvrcp.qrc" >:/images/stop.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ContinuousCheckBox" >
       <property name="text" >
        <string>Continuous</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GroupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget_2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>YUV File:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FilePathEdit" />
     </item>
     <item>
      <widget class="QPushButton" name="BrowseButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>191</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Frame rate:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="FrameRateEdit" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Format:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="FormatCombo" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="dvrcp.qrc" />
 </resources>
 <connections/>
</ui>
